app/models/teacher_training_api/course.rb

```diff
@@ -0,0 +1,18 @@
+module TeacherTrainingAPI
```

Comment:
I wonder if it might pay to be more specific with the module name here (and any other references to the public API throughout the codebase)? 
i.e. `TeacherTrainingPublicAPI` 

Find is still talking to the old V3 API so might prevent a few brain somersaults? 
Publish talks to V2 and UCAS, V1

Reply:
Ha! Yes. I'll rename for maximum explicitness. Everything related to this API will be in this namespace so hopefully that'll give us a fairly strong guarantee we won't mix things up.